src/mpc-hc/resource.h

```diff
@@ -1569,9 +1569,10 @@
 #define IDS_PPAGEADVANCED_SCORE         57530
 #define IDS_PPAGE_FS_CLN_AUDIO_DELAY    57531
 #define IDS_PPAGEADVANCED_DEFAULTTOOLBARSIZE 57532
+#define IDS_SUBTITLE_RENDERER_ASS_FILTER 57533
 
 // Next default values for new objects
-// 

```

Comment:
Please revert this trailing space. It is added by visual studio resource editor. So it needs to stay so it won't pop every time someone edits it.